suite-native/receive/src/screens/ReceiveModalScreen.tsx

```diff
@@ -40,6 +43,15 @@ const ReceiveModalScreenSubHeader = ({ accountKey, tokenContract }: ScreenSubHea
         selectEthereumAccountTokenSymbol(state, accountKey, tokenContract),
     );
 
+    useEffect(() => {
+        const unsubscribe = navigation.addListener('beforeRemove', () => {
```

Comment:
useLayoutEffect did not work?

Reply:
That works as well, good point
https://github.com/trezor/trezor-suite/pull/10250/commits/91a17d64cd1a4c17fe4aac3fa91217bc126f4599

Reply:
https://github.com/trezor/trezor-suite/pull/10250/commits/0577321da19e9e968bdd20d375301b70bff2ef32